files/api/activity.rb

```diff
@@ -33,7 +33,7 @@ def get_events(service: nil, offset: 0, order: 'asc', api_page_size: nil)
         params[:offset] += api_page_size
       end
       raise 'Events API request failed' unless response.code == '200'
-      { 'pagination' => { 'total' => total_count }, 'commits' => response_items }
+      { 'total' => total_count, 'events' => response_items }
```

Comment:
Hey, a small nit. `total` is a little bit ambiguous. Can we call this `api_total_count`, or something like that?